src/AppInstallerCommonCore/AppInstallerCommonCore.vcxproj

```diff
@@ -274,6 +274,21 @@
       <SubSystem Condition="'$(Configuration)|$(Platform)'=='Debug|ARM64'">Windows</SubSystem>
       <SubSystem Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">Windows</SubSystem>
     </Link>
+    <Lib>
+      <AdditionalDependencies Condition="'$(Configuration)|$(Platform)'=='Debug|ARM'">amsi.lib;%(AdditionalDependencies)</AdditionalDependencies>

```

Comment:
Huh, I wonder why we don't have the other dependencies here. I think they are listed in the projects that produce the final binaries, like `WindowsPackageManager.vcxproj`. Maybe we could avoid some duplication by having the dependencies at the point they are introduced...
(Not a comment for this PR)